### DISCUSSION TITLE
Tray icons except microphone disappear when the taskbar is oriented vertically.

### DISCUSSION THREAD

**Discussion by Anixx** [valinet/ExplorerPatcher#382] [General]
![изображение](https://user-images.githubusercontent.com/1271736/141887661-d78af920-9b4b-4b28-8507-babe5f7cf34e.png)
![изображение](https://user-images.githubusercontent.com/1271736/141887687-4ad9eec2-7b42-4545-8946-e1caaa4d7ffe.png)
![изображение](https://user-images.githubusercontent.com/1271736/141887715-7e335dc6-eaf2-4eec-aa65-4e24659fae74.png)
![изображение](https://user-images.githubusercontent.com/1271736/141887746-6aed6820-4cc5-4705-844f-93b206e84858.png)

**Comment by Anixx** [comment]
It seems, this is connected with "fixstrips" action. After first repositioning of the taskbar to vertical position, the icons disappear, but if to drag it to another position and back after this withoput releasing mouse, the icons reappear.
Also, it seems, microphone is not special: only the first icon remains, regardless what it is.

**Comment by Anixx** [comment]
Possibly, this is not fixable until we find a way to get rid of the "fixstrips" hack.

**Comment by valinet** [comment]
We had a thread where we were discussing classic theme related stuff, please move this and future topics regarding classic theme issues there, the way it stands now, it seems it is a thing generally when looking through the discussions list. Plus, one easily knows everything related to the classic theme development is there:

https://github.com/valinet/ExplorerPatcher/discussions/101

Yeah, it's a weird quirk indeed. Maybe because of "fixstrips" (adding `WS_DLGFRAME` style to to `Shell_TrayWnd` and then removing it).

**Comment by Anixx** [comment]
Well, no. The fixstrips is not the culprit, as this happens without classic theme mitigations as well.
